receiver/oracledbreceiver/scraper.go

```diff
@@ -430,6 +451,109 @@ func (s *oracleScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 	return out, nil
 }
 
+func (s *oracleScraper) scrapeLogs(ctx context.Context) (plog.Logs, error) {
+	const dbPrefix = "oracledb."
+	const queryPrefix = "query."
+	const duration = "DURATION_SEC"
+	const event = "EVENT"
+	const hostName = "MACHINE"
+	const module = "MODULE"
+	const osUser = "OSUSER"
+	const objectName = "OBJECT_NAME"
+	const objectType = "OBJECT_TYPE"
+	const process = "PROCESS"
+	const port = "PORT"
+	const program = "PROGRAM"
+	const planHashValue = "PLAN_HASH_VALUE"
+	const sqlID = "SQL_ID"
+	const schemaName = "SCHEMANAME"
+	const sqlChildNumber = "SQL_CHILD_NUMBER"
+	const sid = "SID"
+	const serialNumber = "SERIAL"
+	const status = "STATUS"
+	const state = "STATE"
+	const sqlText = "SQL_FULLTEXT"
+	const username = "USERNAME"
+	const waitclass = "WAIT_CLASS"
+
+	var scrapeErrors []error
+
+	dbClients := s.samplesQueryClient
+	if dbClients == nil {
+		s.logger.Info("Error creating client: ")
+		panic("exiting")
+	}
+	rows, err := dbClients.metricRows(ctx)
+	if err != nil {
+		scrapeErrors = append(scrapeErrors, fmt.Errorf("error executing %s: %w", samplesQuery, err))
+	}
+
+	logs := plog.NewLogs()
+
+	resourceLog := logs.ResourceLogs().AppendEmpty()
+	resourceAttributes := resourceLog.Resource().Attributes()
+	if s.metricsBuilderConfig.ResourceAttributes.OracledbInstanceName.Enabled {
```

Comment:
sorry i may miss something here but i don't quite get why we are adding the attribute based on the metric configuration. i know the current metadata file only support metrics, but i feel it is a little bit confusing to use the metric configuration here.